src/gpu/SDL_gpu.c

```diff
@@ -279,12 +279,19 @@ SDL_GpuComputePipeline *SDL_GpuCreateComputePipeline(
         SDL_InvalidParamError("computePipelineCreateInfo");
         return NULL;
     }
-
+    if (computePipelineCreateInfo->readWriteStorageTextureCount > MAX_COMPUTE_WRITE_TEXTURES) {
+        SDL_assert_release(!"Compute pipeline read-write texture count cannot be higher than 8!");
```

Comment:
Down in BeginComputePass, we're handling this scenario using SDL_InvalidParamError instead of an assert. It feels wrong to do it two different ways, but I guess technically `readWriteStorageTextureCount` isn't a parameter...

If we are going to assert, we should probably do it in the `if (device->debugMode)` block below.